docs/summary.md

```diff
@@ -0,0 +1,10 @@
+# Code review bot for developers
+
+This documentation is targeted towards developers of the code review bot itself, or developers of analyzers who want to integrate it in our bot.
+
+Here is the overall summary of available documentation:
+
+- [How to add a new analyzer on the code review bot](new_analyzer.md)
+- [The default analysis format supported](analysis_format.md)
+
+You can contact the code review bot's developers on Slack #code-review-bot
```

Comment:
Same here, better to avoid duplicating this info, or we need to update it multiple times in different places and we risk forgetting one.